package/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "cmake.configureOnOpen": false
```

Comment:
This file should probably not be committed?

Reply:
definitely not